rest/config_startup.go

```diff
@@ -108,9 +111,10 @@ type APIConfig struct {
 }
 
 type HTTPSConfig struct {
-	TLSMinimumVersion string `json:"tls_minimum_version,omitempty"     help:"The minimum allowable TLS version for the REST APIs"`
-	TLSCertPath       string `json:"tls_cert_path,omitempty" help:"The TLS cert file to use for the REST APIs"`
-	TLSKeyPath        string `json:"tls_key_path,omitempty"  help:"The TLS key file to use for the REST APIs"`
+	TLSMinimumVersion string `json:"tls_minimum_version,omitempty" help:"The minimum allowable TLS version for the REST APIs"`
+	TLSCertPath       string `json:"tls_cert_path,omitempty"       help:"The TLS cert file to use for the REST APIs"`
+	TLSKeyPath        string `json:"tls_key_path,omitempty"        help:"The TLS key file to use for the REST APIs"`
+	UseTLSClient      *bool  `json:"use_tls_client,omitempty"      help:"Allows REST APIs to start without TLS"`
```

Comment:
outdated help text based on the changes to invert this option

```suggestion
	UseTLSClient      *bool  `json:"use_tls_client,omitempty"      help:"Forces the REST APIs to use TLS/HTTPS"`
```